src/picknik_ur_gazebo_config/objectives/apriltag1_grasp_offset.yaml

```diff
@@ -0,0 +1,8 @@
+GraspOffset:
```

Comment:
oh also, should we save defaults for tags 2 and 3 as well so more works out of the box?